Modules/_io/textio.c

```diff
@@ -2395,8 +2392,8 @@ _io_TextIOWrapper_tell_impl(textio *self)
         } \
```

Comment:
I thought that this should be fixed as part of bpo-28261, as bpo31243 is about PyArg_ParseTuple returning 0 or 1.
ISTM that if we add that check for a tuple, we might as well add ";illegal decoder state" to the argument of PyArg_ParseTuple.
Don't you think that leaving it to bpo-28261 is preferable? (I would open a PR for leftovers of bpo-28261 anyway today.)